packages/react/src/runtimes/edge/streams/toolResultStream.ts

```
@@ -1,39 +1,89 @@
+          if (p.type === "tool-call" && p.toolCallId === part.toolCallId) {
+            return {
+              ...p,
+              state: "result" as const,
+              artifact: result.artifact,
+              result: result.result,
+              isError: result.isError,
+            };
+          }
+          return p;
+        });
+        message = {
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)